icp-deploy.tf

```diff
@@ -55,11 +55,11 @@ module "icpprovision" {
       "calico_ip_autodetection_method"  = "first-found"
       "default_admin_password"          = "${var.icppassword}"
       "disabled_management_services"    = [ "${var.va["nodes"] == 0 ? "vulnerability-advisor" : "" }" , "${var.disable_istio == "true" ? "istio" : "" }", "${var.disable_custom_metrics_adapter == "true" ? "custom-metrics-adapter" : "" }" ]
-      "image_repo"                      = "${dirname(local.image)}"
-      "private_registry_enabled"        = "${local.registry_creds != "" ? "true" : "false" }"
-      "private_registry_server"         = "${local.registry_creds != "" ? "${dirname(dirname(local.image))}" : "" }"
-      "docker_username"                 = "${local.registry_creds != "" ? "${replace(local.registry_creds, "/:.*/", "")}" : "" }"
-      "docker_password"                 = "${local.registry_creds != "" ? "${replace(local.registry_creds, "/.*:/", "")}" : "" }"
+      #"image_repo"                      = "${dirname(local.image)}"
+      #"private_registry_enabled"        = "${local.registry_creds != "" ? "true" : "false" }"
```

Comment:
If we're disabling the private_registry code we probably need to clean up the locals for that code, since it's no longer in use....